src/gui/XournalView.cpp

```diff
@@ -24,6 +25,14 @@
 #include "XournalppCursor.h"
 #include "filesystem.h"
 
+std::pair<size_t, size_t> XournalView::preloadPageBounds(size_t page, size_t maxPage) {
+    static const size_t preloadBefore = this->control->getSettings()->getPreloadPagesBefore();
+    static const size_t preloadAfter = this->control->getSettings()->getPreloadPagesAfter();
+    const size_t lower = page > preloadBefore ? page - preloadBefore : 1;
```

Comment:
Do page numbers really start with 1? Awkward... In IT all enumerations normally start at 0.

Reply:
Oops, that's a typo.